molo/yourtips/migrations/0006_auto_20170710_1129.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-07-10 09:29
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
@Mitso: This migration need to be removed, you will have to pull develop and merge that back in. There shouldn't be any new migrations